ModelViewer/ModelViewer/ViewerGraphicsWindow.cpp

```diff
@@ -57,6 +57,136 @@ bool ViewerGraphicsWindow::loadModel(QString filepath) {
     return m_currentModel.m_isValid;
 }
 
+bool ViewerGraphicsWindow::loadVertexShader(QString vertfilepath)
+{
+    if (!initialized)
+    {
+        return false;
+    }
+    if (vertfilepath.isEmpty()) {
+        vertfilepath = QFileDialog::getOpenFileName(nullptr, "Load Vertex Shader", "../Data/Shaders/", "");
```

Comment:
If you do `QFileDialog::getOpenFileName(nullptr, "Load Vertex Shader", "../Data/Shaders/", "*.vert");`, it will only show .vert files.  This might be nice.

Reply:
yeah, I wasn't sure whether we wanted to support other text file formats or not. But that would be fine